app/views/endemics/check-details.njk

```diff
@@ -1,4 +1,5 @@
 {% extends './layouts/layout.njk' %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```

Comment:
Can we move this to the _layout.njk and also remove it from the endemics/declaration page?

Reply:
Done